app/views/pages/cookies.html.erb

```diff
@@ -0,0 +1,131 @@
+<div class="grid-row">
+  <div class="column-two-thirds">
+
+    <header class="text">
+      <h1 class="form-title heading-large">
+        <%= page_title t('.heading_h1') %>
+      </h1>
+    </header>
+
+    <p class="lede">
+      <%= t(".intro") %>
+    </p>
+
+     <p>These cookies are used to:</p>
+
+     <ul class="list-bullet">
+       <li>help us understand how you use the service, so we can make improvements</li>
+       <li>remember what notifications you’ve seen so that you’re not shown them more than once</li>
+       <li>temporarily store the information you enter</li>
+     </ul>
+
+     <p>Find out <a rel="external" href="https://ico.org.uk/for-the-public/online/cookies/">how to manage cookies on the Information Commissioners' website</a>.</p>
+
+     <h2 class="heading-medium">Google Analytics cookie</h2>
+
+     <p>We use Google Analytics to collect information about how you use the service. This information helps us to improve the service.</p>
+
+     <p>The Google analytics cookie collects and stores information about:</p>
+
+     <ul class="list-bullet">
+       <li>the pages you visit - how long you spend on each page</li>
+       <li>how you got to the service</li>
+       <li>what you click on while you’re using the service</li>
+     </ul>
+
+     <table class="cookies-table" role="presentation">
+       <caption class="visually-hidden">
+          Google analytics cookie details
+       </caption>
+       <thead>
+         <tr>
+           <th>Name</th>
+           <th>Purpose</th>
+           <th>Expires</th>
+         </tr>
+       </thead>
+       <tbody>
+         <tr>
+           <td>_ga</td>
+           <td>Used to identify unique users and inform referring sites, visitor and session counts</td>
+           <td>2 years</td>
+         </tr>
+       </tbody>
+     </table>
```

Comment:
I wonder if a [description list](http://www.w3schools.com/tags/tag_dl.asp) would be a better way of  displaying this information. More semantic - see https://css-tricks.com/utilizing-the-underused-but-semantically-awesome-definition-list/